client/windows/CreaturePurhaseCard.cpp

```diff
@@ -15,21 +15,43 @@
 #include "../../CCallback.h"
 #include "../CreatureCostBox.h"
 #include "QuickRecruitmentWindow.h"
+#include "../gui/CGuiHandler.h"
 
 void CreaturePurhaseCard::initButtons()
 {
 	initMaxButton();
 	initMinButton();
+	initCreatureSwitcherButton();
 }
 
 void CreaturePurhaseCard::initMaxButton()
 {
-	maxButton = std::make_shared<CButton>(Point(pos.x + 52, pos.y + 178), "iam014.def", CButton::tooltip(), std::bind(&CSlider::moveToMax,slider), SDLK_m);
+	maxButton = std::make_shared<CButton>(Point(pos.x + 52, pos.y + 178), "iam014.def", CButton::tooltip(), std::bind(&CSlider::moveToMax,slider));
```

Comment:
`initMaxButton` and `initMaxButton` looks like 99% code duplication, please consider refactoring as you already tweaking them.

Reply:
Huh, just want delete SDLK_m shortcut at end :). By mistake, I put it once time and now want remove.

Reply:
BTW, could we add proper shortcuts for that, f.e. {CTRL|SHIFT] + <number> to min|max card <number>?

Reply:
seems nice, done!